Dockerfile.assets-builder

```diff
@@ -1,59 +1,27 @@
-# Debian oldoldstable, support ends in May 2018
-# we need this for Ruby 1.9.x at the moment
-FROM node:6.13.1-wheezy
+ARG image_tag=latest
+ARG node_version
+FROM elifesciences/pattern-library_npm:${image_tag} as npm
+FROM node:${node_version}
 
-# TODO: base image
-USER root
-RUN useradd -ms /bin/bash -G www-data elife && \
-    chown elife:elife /srv && \
-    mkdir /srv/bin && \
-    chown elife:elife /srv/bin && \
-    mkdir -p /var/www && \
-    chown www-data:www-data /var/www
-ENV PATH=/srv/bin:${PATH}
-# end base image
-
-# potentially base image
-RUN npm install -g gulp-cli
-# end potentially base image
-
-RUN apt-get update && apt-get install -y \
-    bzip2 \
-    g++ \
-    make
```

Comment:
Where have `bzip2`, `g++` and `make` gone? Not necessary anymore, what they were used for?

Reply:
The `node` image extends `buildpack-deps` which already contains them.